examples/basic_cuda_example.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Accelerated video decoding with NVDEC
```

Comment:
Users who don't know what NVDEC is (i.e. 99% of them) won't know that this is about CUDA GPUs. I think the title should include the word "CUDA" and / or "GPU". Maybe

> Accelerated video decoding on CUDA GPUs with NVDEC

?

Reply:
I updated this now

Reply:
Agreed that we want a title that people who are not familiar with NVDEC will know right away it's about GPUs and CUDA. My suggested phrasing:

> Accelerated video decoding on GPUs with CUDA and NVDEC

We also want to define NVDEC in our tutorial. We can potentially link to some NVIDIA docs for readers to learn more, such as: https://docs.nvidia.com/video-technologies/video-codec-sdk/12.1/nvdec-video-decoder-api-prog-guide/index.html.